releasenotes/notes/otlp-log-ingestion-config-6da2fca24efe1fad.yaml

```
@@ -0,0 +1,3 @@
+features:
+  - |
+    Add config variables and environment variable ``DD_OTLP_CONFIG_LOGS_ENABLED `` related to OTLP log ingestion
```

We should list the new config name instead of the env vars and explain it's behavior.

WDYT of:
```suggestion
    Adding new configuration setting ``otlp_config.logs.enabled`` to enable/disable logs support  in the OTLP ingest endpoint.
```